Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,48 @@
 import React from 'react';
 import './App.css';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className='container'>
+
+      <div className='row'>
+        <CalculatorDisplay />
+      </div>
+
+      <div className='row'>
+      <ActionButton text="clear"/>
+      <NumberButton buttonStyle="red-button" text="÷"/><br />
```

Comment:
Excellent work organizing the calculator components and hard coding the symbols!